### ISSUE TITLE
a speed problem

### ISSUE THREAD

**Issue by Elvatofeo** [SheerSt/pokewilds#89]
Hi, my computer isn't bad, it runs other games well, but specifically this one, it doesn't run well, it's slow. What can I do to make it go faster?

**Comment by Enzi-Sloh** [comment]
Yeah I have this problem too but only when my game is full-screen because it's loading a large portion of the game at once. If you make the game screen smaller it should run normally, if your problem is the same as mine that is.

**Comment by Elvatofeo** [comment]
I've already tried; sometimes it works but sometimes it doesn't. Isn't it a problem with the game itself?

**Comment by SheerSt** [comment]
@Sam-Losh @Elvatofeo I am working on a newer build of v0.8.3 that has some optimizations to make the game run faster. if either of you would like to try the build out, feel free to ping me on discord and I will add you to the testing channel 👍

**Comment by Elvatofeo** [comment]
Hello, thank you very much for everything. How do I add you to Discord? @SheerSt

**Comment by SheerSt** [comment]
@Elvatofeo here is the invite link to the Wilds server: https://discord.gg/zzuVK8MdKK

You should be able to DM once on the server 👍

**Comment by Elvatofeo** [comment]
@SheerSt Thanks

**Comment by fourlastor** [comment]
Closing this as resolved
